src/Hydra/Action/DocumentationAction.php

```diff
@@ -30,15 +30,9 @@ public function __construct(ApiDocumentationBuilderInterface $apiDocumentationBu
 
     /**
      * Gets API doc.
-     *
-     * @param Request $request
-     *
-     * @return array
      */
-    public function __invoke(Request $request)
+    public function __invoke() : JsonResponse
     {
-        $request->attributes->set('_api_format', 'jsonld');
-
-        return $this->apiDocumentationBuilder->getApiDocumentation();
+        return new JsonResponse($this->apiDocumentationBuilder->getApiDocumentation(), JsonResponse::HTTP_OK, ['Content-Type' => 'application/ld+json']);
```

Comment:
I really do not see why we should use the `JsonResponse` class at all. Content negotiation should take place for these requests as well...